Application/EdFi.Ods.Features/Resources/Publishing.json

```diff
@@ -0,0 +1,162 @@
+{
+  "host": "%HOST%",
+  "basePath": "%BASE_PATH%",
+  "swagger": "2.0",
+  "info": {
+    "title": "Publishing Snapshots API Endpoint",
+    "version": "1.0.0",
+    "description":
+    "The Ed-Fi ODS / API enables applications to read and write education data stored in an Ed-Fi ODS through a secure REST interface. *Note: Consumers of ODS / API information should sanitize all data for display and storage. The ODS / API provides reasonable safeguards against cross-site scripting attacks and other malicious content, but the platform does not and cannot guarantee that the data it contains is free of all potentially harmful content.*"
+  },
+  "tags": [
+    {
+      "name": "snapshots",
+      "description": "Gets the available static snapshots for enabling client-side processing that is isolated from changing data."
+    }
+  ],
+  "parameters": {
+    "offset": {
+      "description": "Indicates how many items should be skipped before returning results.",
+      "in": "query",
+      "name": "offset",
+      "required": false,
+      "default": 0,
+      "format": "int32",
+      "type": "integer"
+    },
+    "limit": {
+      "description": "Indicates the maximum number of items that should be returned in the results.",
+      "in": "query",
+      "name": "limit",
+      "required": false,
+      "default": 25,
+      "format": "int32",
+      "maximum": 500,
+      "minimum": 0,
+      "type": "integer"
+    }
+  },
+  "definitions": {
+    "publishing_snapshot": {
+      "properties": {
+        "id": {
+          "description": "A resource identifier that uniquely identifies the resource.",
+          "type": "string"
+        },
+        "snapshotIdentifier": {
+          "description": "The unique identifier of the snapshot to be passed in the 'Snapshot-Identifier' custom header by the API client on subsequent GET requests.",
+          "x-Ed-Fi-isIdentity": true,
+          "maxLength": 32,
+          "type": "string"
+        },
+        "snapshotDateTime": {
+          "description": "The date and time the snapshot was created.",
+          "format": "date-time",
+          "type": "string"
+        }
+      },
+      "required": [
+        "snapshotIdentifier",
+        "snapshotDateTime"
+      ],
+      "type": "object"
+    }
+  },
+  "paths": {
+    "/publishing/snapshots": {
```

Comment:
I think this should only be /snapshots.  The url is ending up as:
"http://localhost:54746/publishing/v1/publishing/snapshots"
instead of
"http://localhost:54746/publishing/v1/snapshots"
which is causing errors when you test it through SwaggerUI.